### PR TITLE
feat(ENG-3730): prevent dropping tables

### PR DESCRIPTION
**Pull request by andrewlaskey**
https://nacelle.atlassian.net/browse/ENG-3730

### WHY are these changes introduced?

We don't want express-cassandra to be able to drop a table even accidentally.

### WHAT is this pull request doing?

Comments out and throws an error for functions that would be used for dropping tables.

Since there are lot of eslint/prettier updates. Here are the lines relevant to table dropping:

- https://github.com/getnacelle/express-cassandra/blob/52e80e8ffb6838feda14b94fad744d19002dfee2/src/orm/base_model.js#L235-L280
- https://github.com/getnacelle/express-cassandra/blob/52e80e8ffb6838feda14b94fad744d19002dfee2/src/builders/table.js#L183-L264
- https://github.com/getnacelle/express-cassandra/blob/fe0b8e0ea196c4d324defe57a4e3024c331fd260/test/functional/migrations.js

### Note

Looks like there are a lot of style updates

### CODE REVIEW

File: test/functional/schema_load_sync.js [hunk 1]
```diff
@@ -245,4 +243,34 @@ module.exports = () => {
       });
     });
   });
+
+  describe('#migrations', () => {
```
Comment (ignacio-badiola) [inline test/functional/schema_load_sync.js:247]: What do you think of add this to a separate test file called `migrations.js` under `test/functional` folder?
Maybe helps to keep separated our migrations tests?

If you agree, then we need to include it as part of the test set:
 https://github.com/getnacelle/express-cassandra/blob/a1c259bdbe31a26fc88b2214a38f556226e6117f/test/test.js#L3

Comment (andrewlaskey) [reply test/functional/schema_load_sync.js:247]: Works for me. I'll make the change.

Comment (ignacio-badiola) [reply test/functional/schema_load_sync.js:247]: Thanks!

**Review by ignacio-badiola — APPROVED** [review]
LGTM! Thanks for adding those changes

**Pull request merged**